src/it/scala/io/iohk/ethereum/txExecTest/ECIP1017Test.scala

```diff
@@ -49,7 +49,8 @@ class ECIP1017Test extends AnyFlatSpec with Matchers {
       ecip1098BlockNumber = Long.MaxValue,
       treasuryAddress = Address(0),
       ecip1097BlockNumber = Long.MaxValue,
-      ecip1099BlockNumber = Long.MaxValue
+      ecip1099BlockNumber = Long.MaxValue,
+      ecip1049BlockNumber = None
```

Comment:
how does this translate to the configuration? if by default the "ecip1049-block-number" is set to 14796 (example) how would I remove it in a child config to get "None" as a result in the code? or is this only for the situation that the parameter was never defined?

Reply:
I think the answer is "this only for the situation that the parameter was never defined", but I am not sure that I understood the question.
At the moment in `etc-chain.conf` the configuration for this ECIP is added but all commented out:
https://github.com/input-output-hk/mantis/blob/feature/ETCM-746-support-keccak256-pow/src/main/resources/conf/chains/etc-chain.conf#L99-L102
That translates to `None`. 

For Astor it will be `ecip1049-block-number = "0"` because it uses Keccak from genesis block and for other networks it should be a specific block number (determining the switch)

Reply:
understood. my point is that the current flow is that if the flag is not set it defaults to Long.MaxValue. maybe changing this flow without addressing the other flags will make the code confusing for other devs? IOW maybe this change can be done as an overall improvement, and avoid this interim where there are mixed flows

Reply:
This flag does not default to `Long.MaxValue`, It defaults to None.

Reply:
that's true. but every other flag of this type defaults to `Long.MaxValue`

Reply:
Ok, I got what you mean. If you agree with me that defaulting to `None` is better than defaulting to `Long.MaxValue` I can create a ticket to address the issue. Or are you thinking an even more deep refactoring?

Reply:
please capture this in a ticket and we can come back to it. I have some ideas but I would also like to hear yours (IMO None isn't the solution but I'm open to a discussion about it).

Reply:
I added ticket ETCM-794